sdk/cognitiveservices/Azure.OpenAI.Inference/tests/OpenAIInferenceTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using NUnit.Framework;
+using Azure.OpenAI.Inference.Models;
+
+namespace Azure.OpenAI.Inference.Tests
+{
+    [TestFixture]
+    public class OpenAIInferenceTests : OpenAITestBase
+    {
+        /// <summary>
+        /// Setup before each unit test
+        /// </summary>
+        [SetUp]
+        public void Init()
+        {
+        }
+
+        /// <summary>
+        /// Test an instance of OpenAIClient
+        /// </summary>
+        [Test]
+        public void InstanceTest()
+        {
+            var client = GetClient();
+            Assert.That(client, Is.InstanceOf<OpenAIClient>());
+        }
+
+        /// <summary>
+        /// Test Completion
+        /// </summary>
+        [Test]
+        public void CompletionTest()
+        {
+            var client = GetClient();
+            CompletionsRequest completionsRequest = new CompletionsRequest(new List<string> { "Hello world", "running over the same old ground" });
```

Comment:
If this is the reason that we introduce the extra constructor that takes a list of string, we should use the collection initializer instead:
```suggestion
            CompletionsRequest completionsRequest = new CompletionsRequest()
            {
            Prompt = { "Hello world", "running over the same old ground" }
            };
```
Please remove the customization code if this is its only purpose.

Reply:
Removed.